go/arrow/util/util_message/types.pb.go

```diff
@@ -0,0 +1,522 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
This file should get added to dev/release/rat_exclude_files.txt since it is a generated file